bindings/matrix-sdk-ffi/src/event.rs

```diff
@@ -128,7 +128,7 @@ pub enum MessageLikeEventContent {
     KeyVerificationDone,
     ReactionContent { related_event_id: String },
     RoomEncrypted,
-    RoomMessage { message_type: MessageType },
+    RoomMessage { message_type: MessageType, in_reply_to: Option<String> },
```

Comment:
Do we have a better type for an FFI event id passed to the other side? If not, can we put "event_id" somewhere in the name of that new field?